test/com/dataengi/crm/identities/daos/CompaniesDAOSpec.scala

```diff
@@ -42,6 +98,27 @@ class CompaniesDAOSpec extends PlaySpecification with AuthenticationContext with
       getByNameResult.isRight === true
     }
 
+    "Add list of companies" in {
+      val company1         = Company(name = "Test")
+      val company2         = Company(name = "Test1")
+      val CompanyList      = List(company1, company2)
+      val addCompanyResult = companiesDAO.add(CompanyList).await()
+      println(s"[companies-dao][add] $addCompanyResult}")
+      addCompanyResult.isRight === true
+
+      val id1            = addCompanyResult.right.get.head
```

Comment:
If `addCompanyResult.value` be empty then we will get error.

Would be better add check begore
```scala
      val companiesDB = addCompanyResult.value

      companiesDB must have size 2

      val id1            = companiesDB.head
      val id2            = companiesDB.last
```